main.go

```diff
@@ -81,6 +81,11 @@ func main() {
 		config.DisabledPage = v
 	}
 
+	if v := os.Getenv("CONTENT_TYPE_BYTE_LIMIT"); len(v) > 0 {
+		a, _ := strconv.Atoi(v)
```

Comment:
might be worth error checking this, otherwise it'll default to 0 for an invalid value

(I know we haven't for others, but afaik the 'broken default' for others works while this one doesn't)